client/js/dashboard.js

```diff
@@ -108,6 +108,8 @@ if (Element && !Element.prototype.matches) {
             console.error(err);
             ErrorIndicator.show();
         });
+    }).catch(function(e) {
+        ErrorIndicator.show();
```

Comment:
Oh, yes, I see. So a unified error handler could get errors from (1) `Kernel.start` (in which case it has been logged) ; or (2) `widgetsReady` (where it has not been logged).

I think it makes the most sense to remove logging from `Kernel.start`, since it is also throwing the error. It should probably just create a new error with the text `Failed to create kernel`, wrapping the incoming error. Make sense?